app/main.py

```diff
@@ -1 +1,57 @@
-#  write your code here
+from __future__ import annotations
+
+
+class Animal:
+    def __init__(
+            self,
+            name: str,
+            appetite: int,
+            is_hungry: bool = True
+    ) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> None:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> int:
+        if self.is_hungry:
+            result = self.appetite
+            print(f"Eating {self.appetite} food points...")
+            self.is_hungry = False
+            self.appetite = 0
```

Comment:
You don’t need to set appetite to zero, it should be unchangeable. As a result, you don’t need 20 line